index.html

```diff
@@ -852,11 +852,11 @@ <h2>Concrete Lifecycle Example</h2>
   "issuer": "https://example.edu/issuers/565049",
   <span class='comment'>// when the credential was issued</span>
   "issuanceDate": "2010-01-01T19:73:24Z",
-  <span class='comment'>// claims about the subject of the credential</span>
+  <span class='comment'>// claims about the subjects of the credential</span>
   "credentialSubject": {
-    <span class='comment'>// identifier for the subject of the credential</span>
+    <span class='comment'>// identifier for the first subject of the credential</span>
     "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    <span class='comment'>// assertion about the subject of the credential</span>
+    <span class='comment'>// assertion about the first subject of the credential</span>
```

Comment:
perhaps use "only" instead of "first"
this would keep it consistent with your changes elsewhere (and this credential doesn't have more than one subject, so saying "first" may be confusing).

Reply:
OK done